pkg/images/images.go

```diff
@@ -57,7 +57,7 @@ func GenerateRegSyncFile() {
 		}
 		images, err := getImages(distro, versions)
 		if err != nil {
-			logrus.Fatalf("failed to get images for %s: %v", distro, err)
+			//logrus.Fatalf("failed to get images for %s: %v", distro, err)
```

Comment:
Question: Why are this and a few others skipped?

Reply:
I used my registry hence I had to comment this line. But now I am using rancher registry so reverted this comment.